ivy/functional/ivy/manipulation.py

```diff
@@ -365,15 +370,75 @@ def reshape(
     ret
         Reshaped array.
 
-    Examples
-    --------
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.manipulation_functions.reshape.html>`_ # noqa
+    in the standard. The descriptions above assume an array input for simplicity, but
+    the method also accepts :code:`ivy.Container` instances in place of
+    :code:`ivy.Array` or :code:`ivy.NativeArray` instances, as shown in the type hints
+    and also the examples below.
```

Comment:
Hi 🙂, the text here is not required. Would you mind removing this?

Reply:
Gotcha @Aarsh2001 , removed :)